pom.xml

```diff
@@ -188,6 +189,13 @@
             <scope>provided</scope>
         </dependency>
 
+        <!-- Added to override outdated version in mule-service-http -->
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.11.0</version>
```

Comment:
scope test ?

Reply:
Provided scope you mean? The dependency appears twice, one with provided and one with test @giss-ignacio

Reply:
ok. Then it's ok. But you have to add the exclusion to that mule-service-http dependency. 
```
<exclusions>
    <exclusion>
        <groupId>commons-io</groupId>
        <artifactId>commons-io</artifactId>
    </exclusion>
</exclusions>
```